src/index.js

```diff
@@ -198,26 +272,25 @@ exports.init = (Implementation) => {
   new SessionRoute(app, opts).perform();
 
   // Init
-  return buildSchema()
-    .then((models) => {
-      let directorySmartImplementation;
+  try {
+    const models = await buildSchema();
 
-      if (opts.configDir) {
-        directorySmartImplementation = path.resolve('.', opts.configDir);
-      } else {
-        directorySmartImplementation = `${path.resolve('.')}/forest`;
-      }
+    let directorySmartImplementation;
 
-      if (fs.existsSync(directorySmartImplementation)) {
-        return requireAllModels(directorySmartImplementation);
-      }
-      if (opts.configDir) {
-        logger.error('The Forest configDir option you configured does not seem to be an existing directory.');
-      }
+    if (opts.configDir) {
+      directorySmartImplementation = path.resolve('.', opts.configDir);
+    } else {
+      directorySmartImplementation = `${path.resolve('.')}/forest`;
+    }
+
+    if (fs.existsSync(directorySmartImplementation)) {
+      await requireAllModels(directorySmartImplementation);
+    }
+    if (opts.configDir) {
+      logger.error('The Forest configDir option you configured does not seem to be an existing directory.');
+    }
```

Comment:
As we remove the return we should put a `else if` here ;) 
```suggestion
    if (fs.existsSync(directorySmartImplementation)) {
      await requireAllModels(directorySmartImplementation);
    } else if (opts.configDir) {
      logger.error('The Forest configDir option you configured does not seem to be an existing directory.');
    }
```